### ISSUE TITLE
Extend Input component for specific cases

### ISSUE THREAD

**Issue by bafonins** [TheThingsNetwork/lorawan-stack#2259]
#### Why do we need this?

1. **Addresses**, **urls**. We should prefill parts that we are certain about and force the user to fill it in.
<img width="833" alt="Screenshot 2020-03-29 at 19 05 21" src="https://user-images.githubusercontent.com/16374166/77854118-ef5a8c80-71f0-11ea-9add-769759f235d3.png">
<img width="833" alt="Screenshot 2020-03-29 at 19 05 42" src="https://user-images.githubusercontent.com/16374166/77854119-eff32300-71f0-11ea-9231-045d8b5c1a4a.png">
<img width="833" alt="Screenshot 2020-03-29 at 19 05 50" src="https://user-images.githubusercontent.com/16374166/77854120-f08bb980-71f0-11ea-9cea-e1eef3eba84d.png">

Can become (`mqtts://` and `nats.` can be added in the same way):
<img width="763" alt="Screenshot 2020-03-29 at 19 09 35" src="https://user-images.githubusercontent.com/16374166/77854124-fb464e80-71f0-11ea-95f0-256ddfe28400.png">

2. **Units**. We need support for adding custom suffix to start/end of the input component. With such approach, we can communicate units (frequency, time) within the input and not in description/payload.

<img width="223" alt="Screenshot 2020-03-29 at 19 17 09" src="https://user-images.githubusercontent.com/16374166/77854312-3dbc5b00-71f2-11ea-971b-5097daa6a986.png">
<img width="702" alt="Screenshot 2020-03-29 at 19 18 56" src="https://user-images.githubusercontent.com/16374166/77854313-3e54f180-71f2-11ea-9e02-ed3077eaafc8.png">

3. Simple way to add just icons/clickable icons/buttons/links/tooltips to the input, see for example: https://github.com/TheThingsNetwork/lorawan-stack/issues/1095

#### What is already there? What do you see now?

None of the mentioned cases are covered, except for button as action in the input component. 

#### What is missing? What do you want to see?

`<Input />` component being more flexibile
Design guidelines

#### Environment

v3.7.0-rc2

#### Can you do this yourself and submit a Pull Request?

yes

cc @kschiffer

**Comment by kschiffer** [comment]
> Addresses, urls. We should prefill parts that we are certain about and force the user to fill it in.

Agreed, let's add that. Note that such an input should include functionality to strip/replace the given prefix when it is pasted via the clipboard.

> Units. We need support for adding custom suffix to start/end of the input component. With such approach, we can communicate units (frequency, time) within the input and not in description/payload.

This has been added already with the `<UnitInput />` by @pgalic96 via #2113
